agave_app/qtControls/controlFactory.cpp

```diff
@@ -0,0 +1,113 @@
+#include "controlFactory.h"
+
+#include "Controls.h"
+
+QNumericSlider*
+create(const FloatSliderSpinnerUiInfo* info, std::shared_ptr<prtyProperty<float>> prop)
+{
+  QNumericSlider* slider = new QNumericSlider();
+  slider->setStatusTip(QString::fromStdString(info->statusTip));
+  slider->setToolTip(QString::fromStdString(info->toolTip));
+  slider->setRange(info->min, info->max);
+  slider->setDecimals(info->decimals);
+  slider->setSingleStep(info->singleStep);
+  slider->setNumTickMarks(info->numTickMarks);
+  slider->setSuffix(QString::fromStdString(info->suffix));
+
+  slider->setValue(prop->get(), true);
+  QObject::connect(slider, &QNumericSlider::valueChanged, [slider, prop](double value) { prop->set(value, true); });
+  // TODO how would this capture the "previous" value, for undo?
+  QObject::connect(slider, &QNumericSlider::valueChangeCommit, [slider, prop]() { prop->notifyAll(true); });
```

Comment:
Does this still need to call `notifyAll` if `prtyProperty` already does so on `set`?